plugins/org.polarsys.capella.transition.system2subsystem.crossphases/src/org/polarsys/capella/transition/system2subsystem/crossphases/rules/core/PropertyValueRule.java

```diff
@@ -39,4 +41,12 @@ protected EObject getBestContainer(EObject element, EObject result, IContext con
     }
     return super.getBestContainer(element, result, context);
   }
+
+  @Override
+  protected EObject getDefaultContainer(EObject element, EObject result, IContext context) {
+    if (BlockArchitectureExt.getRootBlockArchitecture(element) instanceof OperationalAnalysis) {

```

Comment:
It can be good to add a junit for the #70 usecases to avoid non regression at some point